README.md

```diff
@@ -170,7 +170,7 @@ conda install pandas
 - [SciPy](http://www.scipy.org): miscellaneous statistical functions
 - [PyTables](http://www.pytables.org): necessary for HDF5-based storage
 - [SQLAlchemy](http://www.sqlalchemy.org): for SQL database support. Version 0.8.1 or higher recommended.
-- [matplotlib](http://matplotlib.sourceforge.net/): for plotting
+- [matplotlib](http://matplotlib.org/): for plotting
 - [statsmodels](http://statsmodels.sourceforge.net/)
```

Comment:
Can you also update this to http://www.statsmodels.org/stable/index.html? Thanks.